### PR TITLE
#4280: Pass old cbid to tilize uninit function

### PR DESCRIPTION
**Pull request by rtawfik01**

### CODE REVIEW

File: tt_eager/tt_dnn/op_library/conv/kernels/conv_bmm_tilize_col_major_out_blocks.cpp [hunk 1]
```diff
@@ -186,12 +187,12 @@ void MAIN {
             unpack_reconfig_data_format_srca(in1_cb_id, in0_pretilize_cb_id);
 
             col_major_to_row_major_init();
-            tilize_in(in0_pretilize_cb_id, in0_subblock_h, in0_block_w, in0_num_subblocks, tilized_in0_cb_id);
+            tilize_in(in0_pretilize_cb_id, in0_subblock_h, in0_block_w, in0_num_subblocks, tilized_in0_cb_id, in1_cb_id);
             row_major_to_col_major_init();
 
             // TODO: unpack_reconfig_data_format_srca(in0_pretilize_cb_id, in1_cb_id) doesn't work if in0 is BFLOATB_B and in1 is BFLOAT16
```
Comment (TT-BrianLiu) [inline tt_eager/tt_dnn/op_library/conv/kernels/conv_bmm_tilize_col_major_out_blocks.cpp:193]: Delete

**Review by TT-BrianLiu — APPROVED** [review]
Minor comment

File: tt_eager/tt_dnn/op_library/conv/kernels/conv_bmm_tilize_col_major_out_blocks.cpp (hunk 1 above)
Comment (rtawfik01) [reply tt_eager/tt_dnn/op_library/conv/kernels/conv_bmm_tilize_col_major_out_blocks.cpp]: Done

**Review by mywoodstock — APPROVED** [review]

**Comment by rtawfik01** [comment]
Will merge since I got a passing pipeline here: https://github.com/tenstorrent-metal/tt-metal/actions/runs/8010130985/job/21884816480

**Pull request merged**